packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5615,20 +5666,170 @@ class Listener extends SingleChildRenderObjectWidget {
   }
 }
 
+/// A widget that calls callbacks in response to mouse events.
```

Comment:
After reading this line I expected this to also handle mouse clicks. Should we be specific here and just say mouse hover events?